toolkit/committee-selection/authority-selection-inherents/src/select_authorities.rs

```diff
@@ -49,7 +49,7 @@ pub fn select_authorities<
 	}
 }
 
-pub fn seed_from_nonce_and_sc_epoch(
+fn seed_from_nonce_and_sc_epoch(
```

Comment:
This function is well defined and I think it is usable for users that would like to have their variations of Ariadne